### PR TITLE
Not ERP but Business solution.

### PR DESCRIPTION
**Pull request by cedk**

### CODE REVIEW

File: theme/templates/index.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
 <section class="characteristics">
   <div class="container">
     <h1 class="text-center"><small>{% trans %}World's Most Stable{% endtrans %}</small><br>
-      <strong>{% trans %}OPEN SOURCE ERP SOLUTION{% endtrans %}</strong></h1>
+      <strong>{% trans %}OPEN SOURCE BUSSINES SOLUTION{% endtrans %}</strong></h1>
```
Comment (sharoonthomas) [inline theme/templates/index.html]: Business

**Comment by tarunbhardwaj** [comment]
@cedk merge all 3 commits to a single commit.

**Comment by cedk** [comment]
I don't understand, is it a request or a statement?

**Comment by tarunbhardwaj** [comment]
Its a request.

**Comment by cedk** [comment]
I don't know how to do it from the web interface.

**Comment by tarunbhardwaj** [comment]
Yes, web interface not support this. But you can do it from terminal.

Ref: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by cedk** [comment]
So the fast contribution argument is just pointless.
PR just sucks compare to Rietveld.

**Comment by tarunbhardwaj** [comment]
Of course anything you do not know always sucks.. 

You can either send a single commit PR or I can rebase and merge this (because it only takes a single command to do it).

_PS: Please keep communication in a PR limited to your patch. I will have to lock this PR (and issue) if you bring in unrelated topics to the patch discussion._

**Comment by cedk** [comment]
Do what ever you want.

**Pull request closed without merging**